articles/managed-operations/man-ref-customer-prereqs.md

```diff
@@ -0,0 +1,124 @@
+
+UKCloud will provide you with the installation *package(s)* and commands to install and configure the antivirus agent on your virtual machines.
+
+## Patching as a Service
+
+Patching as a Service provides routine OS patch management for compute instances. It helps you accelerate business value by removing the burden of basic OS hygiene, letting you focus your attention above the operating system. For more information, see the [*Patching as a Service Service Scope*](man-sco-patching.md).
+
+### Operating systems
+
+UKCloud only supports in-life, vendor-backed operating systems, such as Microsoft Windows, Red Hat Enterprise Linux and Canonical Ubuntu. Customer-built operating systems (for example Gentoo Linux custom build) are unsupported by this service. Virtual software appliances are also unsupported by this service.
+
+### Customer environment configuration
+
+#### Linux
+
+To provide secure communication between UKCloud's centralised patch management platform and your virtual machine, we use key-based SSH sessions, reducing the need to share credentials. As such, you'll need to fulfil the following prerequisites on each virtual machine subscribing to this service:
+
+- An allow rule for port TCP/22 (SSH) on your virtual machine's firewall (for example, iptables).
+
+- A local user account with passwordless sudo access. You'll need to share details of this account with UKCloud when onboarding to this service.
+
+- The `/etc/ssh/sshd_config` configuration file must be configured to allow SSH access for the patching user you created in the step above.
+
+- UKCloud will provide you with a public SSH key during the onboarding process. You'll need to append this key to the `authorized_keys` file, located at `~/.ssh/authorized_keys` within the home directory of the user you created in the step above. As part of the onboarding process, UKCloud can perform this action as long as the above requirements are already met.
+
+#### Windows Server
+
+For UKCloud to correctly scan for and apply operating system patches, you must agree to enable and configure the Server Message Block (SMB) protocol by ensuring ports TCP/445 and TCP/139 are open on all virtual machines and associated operating systems opted-in to this service. Furthermore, there must be a bi-directional rule allowing port TCP/3121 on your edge appliance and operating system firewall. For further details, please see the vendor's documentation found [here](https://forums.ivanti.com/s/article/Agentless-Patch-Scanning-Prerequisites?language=en_US).
+
+Furthermore, you must create a patching user that is an administrator on the virtual machine and has access to the Control Panel. Finally, you must enable the Remote Registry service on all virtual machines and enable Filer and Print Sharing for all networks.
+
+##### Domain joined
+
+If the Windows virtual machines are domain joined, the device must be using domain administrator credentials.
```

Comment:
the user* must be using domain administrator credentials

Reply:
Resolved in 7603474.